lib/schwifty-models/users.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const Model = require('schwifty').Model;
+const Joi = require('joi');
+
+module.exports = (srv, options) => {
+
+    return class Users extends Model {
+
+        static get tableName() {
+
+            return 'users';
+        }
+
+        static get joiSchema() {
+
+            return Joi.object({
+
+                id: Joi.number(),
+                email: Joi.string().email(),
+                password: Joi.string().allow(null),//does objection have a counterpart to 'protected' for this?
```

Comment:
This is something I wound up handling in the route, which we decided was a better way to handle it.  This comment was a leftover from the migration from waterline, where the `password` attribute had a `protected` flag set on it.